Features/Language.feature

```diff
@@ -1,27 +1,65 @@
 ﻿Feature: Mars
 
 A short summary of the feature
+
+Background: Clean all data before adding new data
+Given Delete all before adding new langauge
+
+
+
 Scenario Outline:  A.User able to enter the known languages and levels
-Given User able to signin the Mars portal
-When user able to enter their language<language1><language2><language3><language4>
-Then  verify user can able add more than four values
+Given user able to enter their language<language1><language2><language3><language4> and levels
+Then verify user can able add more than four values
 
 Examples: 
 | language1 | language2   | language3 | language4 |
 | 'Tamil'   | 'Telugu'  | 'English' | 'French' |
 
 
  Scenario Outline: B.User able to update the language and levels
- Given User able to edit the language filed<newlanguage>
- When User able to edit the levels<levels>
+ Given User able to edit the language filed<newlanguage><levels>
  Then  verify the both fields update successfully
 
  Examples: 
  | newlanguage | levels  |
  | 'German'    | 'Basic' |
  
- Scenario:C.User able to delete the record
- Given User able to give cancel from edit 
- Given User able to delete the record
+
+ Scenario Outline:C.User able to delete the record
+Given User able to delete the record<Deletelanguage><levels>
```

Comment:
so you have thought about cleanup and state! what i want you to do is think a little further 

so think about first clean up before any test runs (you delete all data around this feature e.g  before any language test run you delete all languages). Then after each test delete only data that the test has added! (test1 added english and test2 to added french then test1 should only delete english and test2 should delete french only)